pkg/controllers/user/pipeline/controller/pipelineexecution/pipelineexecution.go

```diff
@@ -491,7 +491,7 @@ func (l *Lifecycle) doNotify(obj *v3.PipelineExecution) (runtime.Object, error)
 			notifierMessage.Content = strings.Replace(message, "\n", "<br>\n", -1)
 		}
 		g.Go(func() error {
-			return notifiers.SendMessage(toSendRecipient.Notifier, toSendRecipient.Recipient, notifierMessage)
+			return notifiers.SendMessage(toSendRecipient.Notifier, toSendRecipient.Recipient, notifierMessage, nil)
```

Comment:
Looks like we should use dialer here as well so that notification works in a consistent way.